projects/1.21/assets/rar-compat/rarcompat/lang/zh_cn.json

```
@@ -0,0 +1,95 @@
+{
+  "tooltip.relics.umbrella.description": "缓冲摔落，在任何高度摔下都不会受伤。若是使用得更熟练，还能当做可靠的盾牌，以此挡开各种敌人。",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: Softens the fall, ensuring a safe landing from any height. With skillful use, it can serve as a reliable shield, keeping enemies at bay.**

```suggestion
  "tooltip.relics.umbrella.description": "减缓摔落，从任何高度摔下都不会受伤。若使用得当，还能当做一面可靠的盾牌，让敌人无法靠近。",
```